src/types/interfaces.ts

```diff
@@ -1163,3 +1164,63 @@ export interface IVideoResolution {
   width: ConstrainULong,
   height: ConstrainULong
 }
+
+export interface IMemberStatusMessage extends JsonRpcMessage {
+  method: 'member.notify.status';
+  params: {
+    speakers?: VideoSpeakerStatus[];
+    outgoingStreams?: OutgoingStreamStatus[];
+    incomingStreams?: IncomingStreamStatus[];
+  }
+}
+
+export interface VideoSpeakerStatus {
+  id: string;
+
+  activity: 'speaking' | 'inactive' | 'non-speech';
```

Comment:
This _may_ need to be updated after Kevin starts sending messages to indicate a user has stopped speaking, I don't know what that `activity` will look like.